core/formats/wand_writer.hpp

```diff
@@ -72,9 +72,7 @@ class WandWriterImpl final : public WandWriter {
   }
 
   void WriteRoot(index_output& out) final {
-    auto max = std::max_element(score_levels_.begin(), score_levels_.end());
-    IRS_ASSERT(max != score_levels_.end());
-    ValueProducer::Write(max->value, out);
+    ValueProducer::Write(score_levels_.back().value, out);
```

Comment:
Looks like in order to make WriteRoot work properly one needs to call SizeRoot.
Could we make WriteRoot correctness  not dependable on SizeRoot call?